engine/src/build/toolchain/mac/BUILD.gn

```diff
@@ -143,6 +194,133 @@ template("mac_toolchain") {
           [ "{{source_out_dir}}/{{target_output_name}}.{{source_name_part}}.o" ]
     }
 
+    tool("swift") {
+      _tool = rebase_path("//build/toolchain/apple/swiftc.py", root_build_dir)
+
+      depfile = "{{target_out_dir}}/{{module_name}}.d"
+      depsformat = "gcc"
```

Comment:
For those who haven't been sufficiently traumatised by `gn`, the reason for the seemingly random alternation between variables prefixed/unprefixed in `_` is that the unprefixed ones are special `tool` attributes that `gn` knows about whereas the prefixed ones are locals to make this less painful to read.